datayoga-py/datayoga/jmespath_custom_functions.py

```diff
@@ -41,3 +47,83 @@ def _func_mid(self, element, offset, amount):
     @functions.signature({"types": ["string", "null"], "variadic": True})
     def _func_split(self, element, delimiter=","):
         return str(element).split(delimiter) if element is not None else None
+
+    @functions.signature()
+    def _func_uuid(self):
+        """Generates random uuid4 and returns it as string in standard format."""
```

Comment:
Generates a random UUID4 and returns it as a string in standard format

Reply:
✅